sources/packages/backend/libs/test-utils/src/factories/application.ts

```diff
@@ -140,7 +119,66 @@ export async function saveFakeApplicationDisbursements(
     secondSchedule.disbursementDate = getISODateOnlyString(addDays(60));
     disbursementSchedules.push(secondSchedule);
   }
-  fakeOriginalAssessment.disbursementSchedules = disbursementSchedules;
+  savedApplication.currentAssessment.disbursementSchedules =
+    disbursementSchedules;
+
+  savedApplication.currentAssessment = await studentAssessmentRepo.save(
+    savedApplication.currentAssessment,
+  );
+  return applicationRepo.save(savedApplication);
+}
+
+/**
+ * Create and save to the database an application with all the dependencies.
+ * @param dataSource manages the repositories to save the data.
+ * @param relations dependencies.
+ * - `institution` related institution.
+ * - `institutionLocation` related location.
+ * - `student` related student.
+ * @param options additional options:
+ * - `applicationStatus` application status for the application.
+ * @returns the created application.
+ */
+export async function saveFakeApplication(
+  dataSource: DataSource,
+  relations?: {
+    institution?: Institution;
+    institutionLocation?: InstitutionLocation;
+    student?: Student;
+  },
+  options?: {
+    applicationStatus?: ApplicationStatus;
+  },
+): Promise<Application> {
+  const userRepo = dataSource.getRepository(User);
+  const studentRepo = dataSource.getRepository(Student);
+  const applicationRepo = dataSource.getRepository(Application);
+  const studentAssessmentRepo = dataSource.getRepository(StudentAssessment);
+  const offeringRepo = dataSource.getRepository(EducationProgramOffering);
+  const applicationStatus =
```

Comment:
This default value made sense while it was in the method saveFakeApplicationDisbursements as per the old comment on top of it.
```ts
// Using Assessment as a default status since it the first status when
// the application has the disbursement already generated.
```
I believe that we can move the default value assessment as it was before to the `saveFakeApplicationDisbursements`.